typescript/sdk/src/deploy/core/HyperlaneCoreDeployer.ts

```diff
@@ -173,35 +210,17 @@ export class HyperlaneCoreDeployer<
       multisigIsm.address,
       proxyAdmin,
     );
-    // Mailbox ownership is transferred upon initialization.
-    const ownables: Ownable[] = [multisigIsm, proxyAdmin];
```

Comment:
I don't think I understand why we don't need to transfer ownership for the multisigIsm?

Reply:
because we do it in the constructor